src/main/java/net/rptools/maptool/client/functions/MacroJavaScriptBridge.java

```diff
@@ -147,6 +149,43 @@ public Object childEvaluate(
     }
   }
 
+  public Object HostObjectToJavaScriptType(Object obj) {
+    if (obj instanceof JsonElement) {
```

Comment:
I would prefer to handle the negation first. It reduces indent for this method and improved readability IMO.

Reply:
i could do that however then in the future if it were necessary to add more conversions  it could get messy,as it would need a extra "and" comparison per class that needs conversion. what are the plans for the future? if we want to make javascript a first class scripting language then it is almost certain that adding more conversions would be necessary.

Reply:
I think leave it as is but since we are using newer version of java you can use pattern matching
if (obj instanceof JsonElement jObj) {
...
}

this removes the requirement for the explicit cast in the next line